### PR TITLE
Basic skeleton for problem runner. Resolves #3

### LINKED ISSUE #3: Restructure the repository setup
The current repository is unnecessarily specific for the currently projected ambition and size of the project ("Algebra5-6").

The scope of this task is to:

- [x] Rename the repository from `Algebra5-6` to `qarakusi.am`
- [x] Create a `problems` subdirectory in the root of the repo and initialize it with an `__init__.py`. This directory will contain the individual problem scripts
- [x] Create a stub `qarakusi.am.py` script in the root of the directory that'll be the future command-line runner for the scripts in `problems`

### PR DESCRIPTION
**Pull request by sssilver**

### CODE REVIEW

File: qarakusi.am.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python3
```
Comment (zidder) [inline qarakusi.am.py:1]: Is it possible to do all this in Makefile? Or at least run this from Makefile someway.
Also, I'm not sure if `/usr/bin/env python3` will work on Windows.

File: qarakusi.am.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+import importlib
+import sys
+
+
+def main():
+    if len(sys.argv) != 2:
+        sys.exit('Usage: qarakusi.am.py <problem-number>')
+
+    problem_id = sys.argv[1]
```
Comment (zidder) [inline qarakusi.am.py:11]: Can we use argparse instead?

https://docs.python.org/3/library/argparse.html

Comment (sssilver) [reply qarakusi.am.py:11]: We can, my plan was to replace command-line parsing with a proper command-line library through a separate ticket, in the spirit of keeping individual PR scope small. What do you think about that?

File: qarakusi.am.py (hunk 1 above)
Comment (sssilver) [reply qarakusi.am.py:1]: I'm not fully on board with how we use a Makefile in this project. The fact that all of the target in the Makefile are marked as `.PHONY` is a good indication that we aren't making a very idiomatic use of `make`. However, I'm happy to have that conversation separately.

To alleviate your concerns on platform compatibility with Windows, I ran the script in 3 shells under Windows:

1. cmd.exe:
image

2. PowerShell:
image

3. zsh running in ConEmu:
image

File: qarakusi.am.py (hunk 2 above)
Comment (sssilver) [reply qarakusi.am.py:11]: Filed https://github.com/qarakusi-am/qarakusi.am/issues/6

File: qarakusi.am.py (hunk 1 above)
Comment (zidder) [reply qarakusi.am.py:1]: Great, thanks!

File: qarakusi.am.py (hunk 2 above)
Comment (zidder) [reply qarakusi.am.py:11]: Makes sense

**Review by zidder — APPROVED** [review]

**Pull request merged**